facefusion/choices.py

```diff
@@ -20,6 +20,7 @@
 face_detector_score_range : List[float] = numpy.arange(0.0, 1.05, 0.05).tolist()
 face_mask_blur_range : List[float] = numpy.arange(0.0, 1.05, 0.05).tolist()
 face_mask_padding_range : List[float] = numpy.arange(0, 101, 1).tolist()
+face_mask_options : List[str] = [ 'occluder' ]
```

Comment:
This is the list of ranges - you need to move it up

Should be face_mask_modes : List[str] = [ 'box', 'occluder' ]